content/_changelogs/12.0.0.md

```diff
@@ -1,13 +1,20 @@
-## 11.0.0
+## 12.0.0
```

Comment:
I'll open a follow up PR for the changelog. This is more for notes at this time.